File: C#/Easy/Concatenation of Array.cs
    public class Solution
    {
        public int[] GetConcatenation(int[] nums)
        {
            int[] nums2 = new int[nums.Length * 2];
            for (int i = 0; i < nums.Length; i++)
            {
                nums2[i] = nums[i];
            }
            for (int i = 0; i < nums.Length; i++)
            {
                nums2[nums.Length + i] = nums[i];
            }
            return nums2;
        }
    };

File: Java/Easy/ConcatenationofArray.java
package easy;

public class ConcatenationofArray {
    public int[] getConcatenation(int[] nums) {
      int[] answer = new int[nums.length *2];  
        int i =0;
        for(int number : nums){
       answer[i] = number;
        i++;
      }
        for(int number : nums){
       answer[i] = number;
        i++;
      }
        return answer;
    }
}